src/index.js

```diff
@@ -9,5 +9,39 @@ import './css/base.scss';
 
 // An example of how you tell webpack to use an image (also need to link to it in the index.html)
 import './images/turing-logo.png'
+$( document ).ready(function() {
 
-console.log('This is the JavaScript entry file - your code begins here.');
+$('.login-button').on( "click", function() {
+
+  let userName = $('.user-login').val();
+  let userpassword = $('.password').val();
+
+  if (userName === 'manager' && userpassword === 'overlook2019') {
```

Comment:
Fascinating to see! I tackled mine a little differently (as you'll see) so it's always cool to see how others have approached it.